stage-ovos/02-ovos/files/mycroft.conf

```diff
@@ -1,66 +1,68 @@
-      },
-   "listener": {
-      "mute_during_output": false,
-      "instant_listen": true,
-      "VAD": {
-         "module": "ovos-vad-plugin-webrtcvad",
-         "ovos-vad-plugin-webrtcvad": {"vad_mode": 3}
-         },
-      "retry_mic_init" : false,
-      "wake_word_upload": {
-         "disable": true}
-      },
-   "precise": {
-      "use_precise": false
-      },
-   "hotwords": {
-      "hey_mycroft-precise_lite": {
-         "module": "ovos-ww-plugin-precise-lite",
-         "model": "/home/ovos/.local/share/precise-lite/hey_mycroft.tflite",
-         "listen": true,
-         "sound": "snd/start_listening.wav",
-         "expected_duration": 3,
-         "trigger_level": 3,
-         "sensitivity": 0.5,
-         "fallback_ww": "hey_mycroft_openwakeword"
-      },
-      "hey_mycroft_openwakeword": {
-         "module": "ovos-ww-plugin-openwakeword",
-         "models": ["home/ovos/.local/share/openwakeword/hey_mycroft_v0.1.onnx"],
-         "threshold": 0.3,
-         "custom_verifier_threshold": 0.1
-         },
-      "hey_mycroft_vosk": {
-         "module": "ovos-ww-plugin-vosk",
-         "model_folder": "home/ovos/.local/share/vosk/vosk-model-small-en-us-0.15",
-         "samples": ["hey mycroft", "hey microsoft", "hey mike roft", "hey minecraft"],
-         "rule": "fuzzy",
-         "listen": true,
-         "fallback_ww": "hey_mycroft_pocketsphinx"
-         },
-      "hey_mycroft_pocketsphinx": {
-         "module": "ovos-ww-plugin-pocketsphinx",
-         "phonemes": "HH EY . M AY K R AO F T",
-         "threshold": 1e-90,
-         "lang": "en-us",
-         "listen": true
-         }
-      },
-   "stt": {
-      "fallback_module": ""
-      }
+    "ready_settings": ["skills"],
+    "confirm_listening": true,
+    "play_wav_cmdline": "paplay %1",
+    "play_mp3_cmdline": "mpg123 %1",
+    "ipc_path": "/ramdisk/mycroft/ipc/",
+    "network_tests": {
+        "ip_url": "https://api.ipify.org",
+        "dns_primary": "1.1.1.1",
+        "dns_secondary": "8.8.8.8",
+        "web_url": "http://nmcheck.gnome.org/check_network_status.txt",
+        "web_url_secondary": "https://checkonline.home-assistant.io/online.txt",
+        "captive_portal_url": "http://nmcheck.gnome.org/check_network_status.txt",
+        "captive_portal_text": "NetworkManager is online"
+        },
+    "listener": {
+        "mute_during_output": false,
+        "instant_listen": true,
+        "VAD": {
+            "module": "ovos-vad-plugin-silero",
+            "ovos-vad-plugin-silero": {"threshold": 0.2},
+            "ovos-vad-plugin-webrtcvad": {"vad_mode": 3}
+            },
+        "retry_mic_init" : false,
+        "wake_word_upload": {
+            "disable": true}
+    },
+    "precise": {
+        "use_precise": false
+        },
+    "hotwords": {
+        "hey_mycroft": {
+            "module": "ovos-ww-plugin-openwakeword",
+            "models": ["/home/ovos/.local/share/openwakeword/hey_mycroft_v0.1.onnx"],
```

Comment:
Unless a model is pre-downloaded, this path will not work.  Maybe change it so it downloads the model when it loads

Reply:
I'm fuzzy on this. I'm good with putting in logic that automatically inserts this into our local mycroft.conf when the user loads the openwakeword module... Just want to be sure that this is the way you want the script to behave. Thanks!

Reply:
it is either download it on use, or have the user download one.  I am inclined to let the plugin do it

Reply:
remove the `"models"`  section here and it will auto download a model

Reply:
Got it.  Is this existing functionality or a request we would be making of the plugin authors?

Reply:
it is up to plugin authors to provide auto download etc, we do that in all official ovos plugins (i think)